lighthouse-core/test/lib/stack-packs-test.js

```diff
@@ -22,11 +22,7 @@ describe('stack-packs lib', () => {
     const result = lighthouseStackPacksDep
       .filter(p => !stackPacksLib.stackPacksToInclude.find(p2 => p2.packId === p.id))
       .map(p => p.id);
-    expect(result).toMatchInlineSnapshot(`
-      Array [
-        "joomla",
```

Comment:
lolll

Reply:
ya it didnt work for its intended purpose lol

Reply:
well at least we know it works :)

Reply:
can we convert this to `toEqual([]);` with a comment that we shouldn't bother updating a stack pack until detectors are in place? :)